lib/hocon/impl/simple_config_origin.rb

```diff
@@ -16,7 +16,10 @@ def initialize(description, line_number, end_line_number,
       raise ArgumentError, "description may not be nil"
     end
 
-    @description = description
+    # HACK: naming this variable with an underscore, because the upstream library
+    # has both a member var and a no-arg method by the name "description", and
+    # I don't think Ruby can handle that.
+    @_description = description
```

Comment:
Since things like `==` depend on accessing the instance variable directly we can't remove the `attr_reader`, and if we rename `description()` then we'd have to refactor in a dozen places and it wouldn't match up with the java library. Not sure what else we can do :confused: